### PR TITLE
feat: Create integrators acl schema

### PR DESCRIPTION
**Pull request by rhinodavid**
# Description

Create the schema for the `INTEGRATORS_ACL` configuration. This will replace `API_KEYS_ACL` eventually, but this PR just adds the schema so that it can be reviewed in isolation. The new ACL includes an `integratorId` and has a field for multiple API keys for the integrator.

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [ ] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

File: src/schemas/index.ts [hunk 1]
```diff
@@ -10,14 +11,15 @@ import * as sraOrdersChannelSubscribeSchema from './sra_ws_orders_channel_subscr
 import * as swapQuoteRequestSchema from './swap_quote_request_schema.json';
 
 export const schemas = {
-    sraOrderConfigPayloadSchema,
+    apiKeySchema,
```
Comment (rhinodavid) [inline src/schemas/index.ts:14]: add `integratorsAclSchema` and sort

File: src/schemas/integrator_acl_schema.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "id": "/integratorAclSchema",
+    "properties": {
+        "apiKeys": {
```
Comment (alexkroeger) [inline src/schemas/integrator_acl_schema.json:4]: what's the use case of having multiple keys per integrator?

File: src/schemas/integrator_acl_schema.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            "items": {
+                "type": "string"
+            },
+            "type": "array"
+        },
+        "integratorId": {
+            "type": "string"
+        },
+        "label": {
```
Comment (alexkroeger) [inline src/schemas/integrator_acl_schema.json:13]: is this integrator name?

File: src/schemas/integrator_acl_schema.json (hunk 2 above)
Comment (rhinodavid) [reply src/schemas/integrator_acl_schema.json:4]: migration

File: src/schemas/integrator_acl_schema.json (hunk 3 above)
Comment (rhinodavid) [reply src/schemas/integrator_acl_schema.json:13]: yep.. maybe not the best naming but it's already there https://github.com/0xProject/0x-api/blob/master/src/schemas/api_key_schema.json#L8

File: src/schemas/integrator_acl_schema.json (hunk 2 above)
Comment (alexkroeger) [reply src/schemas/integrator_acl_schema.json:4]: synced over Slack, this will make rolling over more straightforward

**Review by alexkroeger — APPROVED** [review]

**Pull request merged**